policy/handler.go

```diff
@@ -143,7 +203,7 @@ func (h *Handler) Stop() {
 
 // generateEvaluation returns an evaluation if the policy needs to be evaluated.
 // Returning an error will stop the handler.
-func (h *Handler) generateEvaluation(policy *Policy) (*Evaluation, error) {
+func (h *Handler) generateEvaluation(policy *Policy, evalTimestamp int64) (*Evaluation, error) {
```

Comment:
Were you planning on storing this value in the evaluation or is it a leftover from the previous implementation?